[40기 김윤아]Add: startSignUp 페이지 레이아웃 구현

longyuna opened the pull request:
## :: 최근 작업 주제 (하나 이상의 주제를 선택해주세요.)
- [x] 기능 추가
- [x] 리뷰 반영
- [ ] 리팩토링
- [ ] 버그 수정
- [ ] 컨벤션 수정

<br />

## :: 구현 목표 (해당 브랜치에서 구현하고자 하는 하나의 목표를 설정합니다.)
- 회원가입으로 넘어 가기 전 페이지 레이아웃 구현

<br />

## :: 구현 사항 설명 (작업한 내용을 상세하게 기록합니다.)
- 회원가입 전 페이지 레이아웃 구현

<br />


<br />

longyuna committed:
회원가입 초기창 레이아웃 구현완료

longyuna committed:
startSignUp 레이아웃 구현

sodalite1204 reviewed (commented):
윤아님 컴포넌트 이름은 SignIn정도로 해도 될 것 같네요!

longyuna commented:
유진멘토님. 의견 감사합니다. 다만, 저희가 생각한 해당 페이지는 로그인이 아닌 회원가입 페이지로 가기 위한 페이지로 팀원들과 상의해본 결과, signin 보다는 startsignup 또는 signup이라는 워딩이 담긴 다른 단어로 하는 것이 더 좋다고 생각이 들었습니다. 혹시 또 다른 의견이 있으시다면 코멘트 부탁드리겠습니다.

sodalite1204 commented on src/pages/SignUp/StartSignUp.js:
```diff
@@ -0,0 +1,22 @@
+        <div>
+          <span className="startSignUpLoginButton">
+            정육각 회원이신가요?
+            <a className="startSignUpLoginButtonTag">로그인 하기</a>
```
Link 컴포넌트로 변경해주세요!

sodalite1204 commented on src/pages/SignUp/StartSignUp.js:
```diff
@@ -0,0 +1,22 @@
+
+const StartSignUp = () => {
+  return (
+    <div className="startSignUpMain">
```
최상위 클래스 이름 = 컴포넌트 이름의 컨벤션 맞춤면
className="startsignUp"
이 되겠네요!

sodalite1204 commented on src/pages/SignUp/StartSignUp.scss:
```diff
@@ -0,0 +1,42 @@
+    border-radius: 3px;
+    background-color: white;
+    color: #e92d44;
+    font-size: 16px;
```
font-size: 16px은 디폴드 값이여서 굳이 선언해  주지 않아도 됩니다!

sodalite1204 reviewed (commented):
리뷰 확인해주세요!

longyuna committed:
Merge branch 'master' into feature/startsignup
Fix: startSignUP component 명 변경 ( startSignUp -> signUp)
- scss 파일에서 common.scss import
- component 명 변경 (startSignUp -> signUp)

longyuna committed:
Fix: signUp, signUpForm, main 오류 수정
- component 명 변경 (startSignUp -> signUp)
- signUp scss 에 variables.scss import 후 색상 변수 값으로 변경
- signUpForm scss 에 variables.scss import 후 색상 변수 값으로 변경
- main 페이지 BestProduct.js component에서 ul 태그 하단에 오타(;) 삭제
- feature/startsignup branch에서 수정했음

sodalite1204 commented on src/pages/SignUp/SignUp.js:
```diff
@@ -0,0 +1,24 @@
+        <div>
+          <span className="startSignUpLoginButton">
+            정육각 회원이신가요?
+            <a className="startSignUpLoginButtonTag" href="www.naver.com">
+              로그인 하기
+            </a>
```
로그인 하기면 자체 서비스에 로그인 할텐데 왜 href가 naver로 되어있나요?
signin으로 갈 수 있게 라우팅 수정해주세요.

sodalite1204 reviewed (commented):
컴포넌트 이름이 수정되었으면 그에 맞게 className도 전반적으로 수정되어야 할 것 같습니다.

longyuna committed:
Modify: signUp scss 수정
- startSignUpButton font-size 변경
- signUp Link component 연결
- '정육각 회원가입 하기', '로그인 하기' Link component 연결로 이동 가능한 상태로 변경

longyuna committed:
Modify: signUp, sigmUpForm 수정
1. signUp 페이지 className 변경 ( startSignUp -> signUp )
2. signUpForm 페이지 className 변경 ( signUp -> signUpForm )
3. signUpForm 페이지 '이전으로' 버튼 Link 연결

- startSignUp branch에서 수정 완료

sodalite1204 reviewed (approved):
수고하셨습니다 윤아님!
